weaknesses/MASVS-CODE/MASWE-0076.md

```diff
@@ -1,38 +1,79 @@
 ---
 title: Dependencies with Known Vulnerabilities
 id: MASWE-0076
-alias: known-vuln-deps
+alias: dependencies-with-known-vulns
 platform: [android, ios]
 profiles: [L1, L2]
 mappings:
   masvs-v1: [MSTG-CODE-5]
   masvs-v2: [MASVS-CODE-3]
+  android-core-app-quality: [SC-N3, PS-T4]
+  android-risks:
+  - https://developer.android.com/privacy-and-security/risks/insecure-library
+  nist-ssdf: [PS.3.2]
 status: new
 refs:
 - https://developer.android.com/privacy-and-security/risks/insecure-library
+- https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf
+- https://www.ntia.doc.gov/files/ntia/publications/sbom_minimum_elements_report.pdf
+- https://developer.android.com/guide/practices/sdk-best-practices
+- https://developer.android.com/privacy-and-security/security-best-practices#services-dependencies-updated
+- https://developer.android.com/privacy-and-security/security-best-practices#update-dependencies
+- https://support.google.com/googleplay/android-developer/answer/14514531
+- https://support.google.com/googleplay/android-developer/answer/13326895
+- https://developer.apple.com/support/third-party-SDK-requirements/
+- https://www.youtube.com/watch?v=3klmiHX0uVQ
+- https://developer.apple.com/videos/play/wwdc2023/10060/
 ---
 
 ## Overview
 
-"Dependencies with Known Vulnerabilities" are external or third-party libraries, SDKs, or frameworks used by the app that contain publicly documented security flaws, usually through CVEs.
+Mobile apps often depend on third-party libraries, software development kits (SDKs), or frameworks, either open-source components maintained by the community or closed-source products provided by commercial vendors, to implement functionality, streamline development, or integrate platform services.
+
+When these dependencies contain vulnerabilities, they can be more easily exploited than first-party code because these vulnerabilities (and some exploits) are often documented in public databases, such as the CVE list, or accessible through security advisories from maintainers or other Open Source Intelligence (OSINT) sources.
```

Comment:
```suggestion
When these dependencies contain vulnerabilities, they can be more easily exploited than vulnerabilities in first-party code because these vulnerabilities (and some exploits) are often documented in public databases, such as the CVE list, or accessible through security advisories.
```
Since it's already such a long sentence and doesn't really add something?